pkg/scheduler/scheduler.go

```diff
@@ -149,14 +190,19 @@ func (s *Scheduler) schedule(ctx context.Context) {
 			//      evaluated in the same cycle is not taken into account.
 			//   2. An already admitted workload from a different cluster queue is borrowing, since all workloads
 			//      evaluated in the current cycle will compete for the resources that are not borrowed.
```

Comment:
Please adjust the comments to reflect the implementation changes.

Reply:
it is still accurate

Reply:
Added `if they have any common resources assigned`